apps/src/applab/redux/screens.js

```diff
@@ -1,10 +1,29 @@
 import Immutable from 'immutable';
+import {
+  sources as sourcesApi,
+  channels as channelsApi,
+} from '../../clientApi';
 
 const CHANGE_SCREEN = 'screens/CHANGE_SCREEN';
 const TOGGLE_IMPORT_SCREEN = 'screens/TOGGLE_IMPORT_SCREEN';
+const IMPORT = {
+  IMPORT_PROJECT: {
+    START_FETCHING: 'screens/importProject/START_FETCHING',
+    FAILED_FETCHING: 'screens/importProject/FAILED_FETCHING',
+    FINISHED_FETCHING: 'screens/importProject/FINISHED_FETCHING',
+  },
+};
+
+var ImportProjectState = Immutable.Record({
```

Comment:
Do we need the extra level of nesting, i.e. `IMPORT.IMPORT_PROJECT.START_FETCHING` vs. just `IMPORT.START_FETCHING`